LayoutTests/imported/w3c/web-platform-tests/html-aam/roles-contextual.html

```diff
@@ -23,8 +23,9 @@
 
 <!-- el-aside -->
 <nav>
-  <aside data-testname="el-aside-in-section-with-name" data-expectedrole="complementary" aria-label="x" class="ex">x</aside>
+  <aside data-testname="el-aside-in-section-with-name" data-expectedrole="generic" aria-label="x" class="ex">x</aside>
```

Comment:
I think the spec expects `aside`s with accessible names to have a role of `complementary`, so changing the expected role to `generic` isn't correct (based on my reading).

https://w3c.github.io/html-aam/#el-aside

Might be worth addressing this piece in a separate patch (if possible, might not be?)

Reply:
I've rechecked the spec and you're correct. I'll update my patch for the <aside> to have complementary role.